numpy/core/tests/test_umath.py

```diff
@@ -1894,6 +1894,15 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         c = np.add(a, b, out=a)
         assert_(c is a)
         assert_equal(c.info, {'inputs': [0, 1], 'outputs': [0]})
+        # While we're at it, use the same class to check that default output
+        # is never passed on.
+        b = np.sin(a, out=None)
+        assert_('outputs' not in np.sin(a, None).info)
+        assert_('outputs' not in np.sin(a, out=None).info)
+        assert_('outputs' not in np.sin(a, out=(None,)).info)
+        assert_('outputs' not in np.modf(a, None)[0].info)
```

Comment:
Does `np.modf(a, b)` get normalized to `__array_ufunc__(..., out=(b, None))` here?